vars/main.yml

```diff
@@ -25,3 +25,21 @@ __ssh_required_facts:
 # the 'gather_subset' parameter of the 'setup' module
 __ssh_required_facts_subsets: "{{ ['!all', '!min'] +
   __ssh_required_facts }}"
+
+# BEGIN - DO NOT EDIT THIS BLOCK - rh distros variables
+# Ansible distribution identifiers that the role treats like RHEL
+__ssh_rh_distros:
+  - AlmaLinux
+  - CentOS
+  - RedHat
+  - Rocky
+
+# Same as above but includes Fedora
+__ssh_rh_distros_fedora: "{{ __ssh_rh_distros + ['Fedora'] }}"
+
+# Use this in conditionals to check if distro is Red Hat or clone
+__ssh_is_rh_distro: "{{ ansible_distribution in __ssh_rh_distros }}"
+
+# Use this in conditionals to check if distro is Red Hat or clone, or Fedora
+__ssh_is_rh_distro_fedora: "{{ ansible_distribution in __ssh_rh_distros_fedora }}"
```

Comment:
How is this different from `ansible_facts['os_family'] == 'RedHat'`?

Reply:
I added a Q&A section to the pr body/commit message